doc/user-guide/04-scenarios.adoc

```diff
@@ -67,6 +67,123 @@ a single confidential command to let the redirection also apply for 'set -x'.
 See the description in /usr/share/rear/conf/default.conf
 how to set variables for secret values in a confidential way.
 
+== Bootable ReaR rescue initramfs on IBM Z (s390x) under z/VM
+
+If you want to create the rescue initramfs with a (tar) backup stored on NFS
+server, add the following settings to `/etc/rear/local.conf`:
+
+[source,bash]
+----
+OUTPUT=RAMDISK
+OUTPUT_URL=file:///path/to/initramfs
+BACKUP=NETFS
+BACKUP_URL=nfs://NFS.server.IP/path/to/backup
+----
+
+Make sure that the path in `OUTPUT_URL` exists and create the rescue initramfs
+and backup using `rear mkrescue`.
```

Comment:
"rear mkrescue" does not cerate a backup
so use `rear mkbackup` in the described case here, i.e.
"to create the rescue initramfs with a (tar) backup".

Reply:
Another typo...  Thank you, @jsmeix!

Reply:
Addressed in 74cb29a7.